docs/prompting/zero_shot/style_prompting.md

```diff
@@ -1,7 +1,43 @@
 ---
-title: ""
+title: "Style Prompting"
 description: ""
-keywords: ""
+keywords: "prompt, prompting, prompt engineering, llm, gpt, openai, style prompting, model, AI, python, instructor, zero-shot, role prompting"
 ---
 
-[wip]
+Style prompting<sup><a href="https://arxiv.org/abs/2302.09185">1</a></sup> includes a style, tone, or genre in the prompt. The effect can be similar to [role prompting](https://python.useinstructor.com/prompting/zero_shot/role_prompting/).<sup><a href="https://arxiv.org/abs/2406.06608">\*</a></sup>
```

Comment:
Would suggest giving some examples and then some useful tips below.

Hmm I feel like it might be useful at the end to give the user some sample words to use

<img width="389" alt="CleanShot 2024-06-24 at 18 26 30@2x" src="https://github.com/jxnl/instructor/assets/45760326/853100b5-701a-4b65-a7e1-7b0ebb55d8a8">

We can see here that the paper appendix gives a lot of sample words os maybe pick like a few diff ones and throw in a section below thx